plugins/GCodeReader/FlavorParser.py

```diff
@@ -160,7 +160,8 @@ def _calculateLineWidth(self, current_point, previous_point, current_extrusion,
         # Area of the printed line. This area is a rectangle
         Ae = dVe / dX
         # This area is a rectangle with area equal to layer_thickness * layer_width
-        line_width = Ae / layer_thickness
+        # 11/04/2018 - added divide by 2 to obtain visually correct line widths
```

Comment:
I don't think adding the date in this comment will add anything to the readability of this fix; there's always git blame. I am much more interested in knowing why dividing by 2 is the correct thing to do here (is it a radius vs diameter thing?)

Reply:
> I am much more interested in knowing why dividing by 2 is the correct thing to do here (is it a radius vs diameter thing?)

I guess it could be. I think what is happening is that without the div by 2 the limit test below is coming out true and so the width gets set to 0.35? which would explain why the walls and the infill lines have the same width and everything is too narrow.